iphone/cli/locales/en.js

```diff
@@ -274,5 +274,7 @@
 	"Connected iOS Devices": "Connected iOS Devices",
 	"Type": "Type",
 	"iOS Version": "iOS Version",
-	"CPU Architecture": "CPU Architecture"
+	"CPU Architecture": "CPU Architecture",
+	"for %s builds, exports archive into an ipa and places it on %s": "for %s builds, exports archive into an ipa and places it on %s",
```

Comment:
There's no need to update the locale file. It's so out-of-date, it doesn't really matter.